Source/WebCore/platform/graphics/ImageUtilities.h

```diff
@@ -44,5 +47,8 @@ WEBCORE_EXPORT Vector<String> findImagesForTranscoding(const Vector<String>& pat
 // happens while transcoding, a null string will be added to the returned list.
 WEBCORE_EXPORT Vector<String> transcodeImages(const Vector<String>& paths, const String& destinationUTI, const String& destinationExtension);
 
+WEBCORE_EXPORT std::pair<String, Vector<IntSize>> utiAndAvailableSizesFromImageData(std::span<const uint8_t>);
+WEBCORE_EXPORT bool mayBeSVGData(const Settings&, std::span<const uint8_t>);
```

Comment:
I think maybe is an adverb that means "possibly" or "perhaps". So I think it should treated as one word.

Reply:
Updated.